### PR TITLE
fix: add missing breaker box update in bpm

### LINKED ISSUE #87: Deploy new contracts and execute MU03 governance script on Baklava
**Description**

We need to execute the constant sum pricing module deployment script across all networks including mainnet.

**Acceptance Criteria**
- [x] All deployments are run on all networks
- [x] Execute governance script of Baklava
- [x] Run fork tests

### PR DESCRIPTION
**Pull request by philbow61**
### Description

We missed updating the BreakerBox address in the biPoolManager in the MU03 deployment script. This PR adds the transaction plus a check in the deployment checks.

### Other changes

- The latest broadcast files which should have been part of #104 

### Tested

- Ran a partial script to update the breakerBox address in the BPM
- Ran the deployment checks + fork tests against Baklava

### Related issues

- Fixes #87 

### Backwards compatibility

-

### Documentation

-

### CODE REVIEW

File: script/upgrades/MU03/MU03.sol [hunk 1]
```diff
@@ -179,6 +193,15 @@ contract MU03 is IMentoUpgrade, GovernanceScript {
         abi.encodeWithSelector(Proxy(0)._setImplementation.selector, contracts.deployed("SortedOracles"))
       )
     );
+
+    // Set new BreakerBox address in SortedOracles
+    transactions.push(
```
Comment (ninocomputer) [inline script/upgrades/MU03/MU03.sol:198]: ditto

File: script/upgrades/MU03/MU03.sol [hunk 2]
```diff
@@ -142,21 +144,33 @@ contract MU03 is IMentoUpgrade, GovernanceScript {
...
+        ICeloGovernance.Transaction(
+          0,
+          partialReserveProxy,
+          abi.encodeWithSelector(IReserve(0).addCollateralAsset.selector, bridgedEUROC)
+        )
+      );
+    }
+  }
+
+  function proposal_updateBiPoolManagerImplementation() public {
```
Comment (ninocomputer) [inline script/upgrades/MU03/MU03.sol:159]: I don't think we should set BiPoolManager implementation and BreakerBox in the same function. 
```suggestion
  function proposal_updateBiPoolManagerImplementation() public {
```

**Review by ninocomputer — CHANGES_REQUESTED** [review]

File: script/upgrades/MU03/MU03.sol (hunk 1 above)
Comment (nvtaveras) [reply script/upgrades/MU03/MU03.sol:201]: updated

File: script/upgrades/MU03/MU03.sol (hunk 2 above)
Comment (nvtaveras) [reply script/upgrades/MU03/MU03.sol:160]: updated

**Review by ninocomputer — APPROVED** [review]
much better

**Review by bayological — APPROVED** [review]

**Review by bowd — APPROVED** [review]

**Pull request merged**